noir/compiler/wasm/src/compile.rs

```diff
@@ -33,28 +32,25 @@ export type CompiledContract = {
     name: string;
     functions: Array<any>;
     events: Array<any>;
+    file_map: Record<number, any>;
+    warnings: Array<any>;
 };
 
 export type CompiledProgram = {
     noir_version: string;
     abi: any;
     bytecode: string;
-}
-
-export type DebugArtifact = {
-    debug_symbols: Array<any>;
+    debug_symbols: any;
     file_map: Record<number, any>;
     warnings: Array<any>;
-};
+}
 
 export type CompileResult = (
     | {
         contract: CompiledContract;
-        debug: DebugArtifact;
```

Comment:
As we're pulling the warnings out of the artifact, we'll need to include them on the `CompileResult` so that they can be displayed in JS